tests/InputFiles/Cce/CharacteristicExtract.yaml

```diff
@@ -24,73 +29,117 @@ ResourceInfo:
 
 Observers:
   VolumeFileName: "CharacteristicExtractUnusedVolume"
+  # The reduction file is where the CCE output will be written.
+  # Specifically, it will be in a `/SpectreR0257.cce` where the number is the
+  # ExtractionRadius specified below.
   ReductionFileName: "CharacteristicExtractReduction"
 
 EventsAndTriggers:
+  # Write the CCE time step every Slab. A Slab is a collection of time steps,
+  # where the time step size can be adjusted dynamically.
```

Comment:
This reads as if the slab size is variable since the step sizes are adjusted dynamically. What about

```yaml
# Write the CCE time step every Slab. A Slab is a fixed length of simulation time
# and is not influenced by the dynamically adjusted step size.
```